pythonFiles/pyvsc-run-isolated.py

```diff
@@ -4,14 +4,20 @@
 if __name__ != "__main__":
     raise Exception("{} cannot be imported".format(__name__))
 
+import os
 import os.path
 import runpy
 import sys
 
-# We "isolate" the script/module (sys.argv[1]) by
-# replacing sys.path[0] with a dummy path and then sending the target
-# on to runpy.
-sys.path[0] = os.path.join(os.path.dirname(__file__), ".does-not-exist")
+
+def normalize(path):
+    return os.path.normcase(os.path.normpath(path))
+
+
+# We "isolate" the script/module (sys.argv[1]) by removing current working
+# directory or '' in sys.path and then sending the target on to runpy.
+cwd = normalize(os.getcwd())
```

Comment:
I feel like I'm missing context on the run-isolated change. What is this for?

Reply:
Nothing related. I was hitting an error related to executing 'import ipykernel'. I updated this file while trying to fix that. And while that wasn't it, I don't see why not keep this change, it just normalizes paths.

Reply:
Personally I would prefer to have a separate PR for such changes.
Helps with simpler reviews, track changes & revert commits as well.

Reply:
Yup, this run-isolated file can get pretty funky in some cases, so I think we should consider this as a separate fix. But the interactiveBase part all looks good to me.